moveit_core/robot_state/src/robot_state.cpp

```diff
@@ -1919,6 +1919,9 @@ double RobotState::computeCartesianPath(const JointModelGroup* group, std::vecto
   traj.clear();
   traj.push_back(RobotStatePtr(new RobotState(*this)));
 
+  // For each waypoint we do not use random restarts since they would very likely result in IK jumps
+  std::size_t ik_attempts = 1;
```

Comment:
I don't see a benefit in introducing a new (constant!) variable here. This even more suggests that the number of IK attempts could be changed. I submitted an alternative comment.